schemas/2018-05-01/subscriptionDeploymentTemplate.json

```diff
@@ -399,6 +399,10 @@
             "copy": {
               "$ref": "#/definitions/resourceCopy"
             },
+            "scope": {
+              "type": "string",
+              "description": "Scope property to be used to provide scope for a resource or to be used to specify a scope for Management Group or Tenant level deployments."
```

Comment:
Do we have documentation published on how to use this? This message is going to show up in VSCode when you mouse over the property, so just want to make sure the expected usage is example; I'm not sure I'd be able to figure it out if just presented with this description.

Reply:
What effect does it have on deploymen?

Reply:
For the `microsoft.resources/deployments` type, it controls the target scope of the deployment. Here is an example:
https://github.com/ajf214/personal-arm-templates/blob/master/ARM-Templates/testing/nested-with-scope-property.json

It can also be used for extension resources, but I don't have an example of that handy, and I don't think it's documented yet. @tfitzmac will be working on that this week.

Reply:
@alex-frankel It doesn't look like it should have any effect on the extension?  E.g., no effect on the syntax or semantics of resources or expressions, etc.  Correct?  thx.

Reply:
I think there will be false negatives if the scope property is used on resources that don't support it.  E.g. if I use it on a website or vm or other tracked resource type. @filizt / @anthony-c-martin do we have an accurate list of all the extension resources? We may need to create some sort of whitelist

Reply:
See https://github.com/Azure/azure-resource-manager-schemas/blob/master/generator/autogenlist.ts - anything marked with `ScopeType.Extension`. Currently these are completely omitted from the root schemas, so we may need to do something special to support them.

Reply:
@anthony-c-martin is there any way to know if a resource type is an extension resource by using the schemas in this repo? The only way I know how to tell that is through service manifest where the routing type is defined.

Reply:
For teams not onboarded for autogeneration, I believe extension resources are not output at all. I'm actually curious what the current authoring experience looks like for extension resources...